pages/common/dep.md

```diff
@@ -0,0 +1,23 @@
+# dep
+
+> Tool for dependency management in the Go ecosystem.
+
+- Initialize a directory to be a project:
+
+`dep init`
+
+- Install the project dependencies:
+
+`dep ensure`
```

Comment:
The description is fine, but just so it matches the actual command word, maybe "Install any missing dependencies of the project". Also, I assume this only checks for Go dependencies, right? In which case, I'd specify that as well ("any missing Go dependencies", or another wording that you find more sensible).

Reply:
Yes. I will change the wording.